qiskit_machine_learning/neural_networks/effective_dimension.py

```diff
@@ -0,0 +1,354 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An implementation of the effective dimension algorithm."""
+
+import time
+from typing import Optional, Union, List, Callable, Tuple
+
+import numpy as np
+from scipy.special import logsumexp
+
+from qiskit.utils import algorithm_globals
+from qiskit_machine_learning import QiskitMachineLearningError
+
+from .opflow_qnn import OpflowQNN
+from .neural_network import NeuralNetwork
+
+
+class EffectiveDimension:
+    """
+    This class computes the global effective dimension for Qiskit
```

Comment:
```suggestion
    This class computes the global effective dimension for a Qiskit
```
This is how it currently reads
> This class computes the global effective dimension for Qiskit NeuralNetwork following the definition used in [1]

So just adding the `a` so it reads a little easier
>This class computes the global effective dimension for a Qiskit NeuralNetwork fo...